docs/source/datasets/index.rst

```diff
@@ -20,7 +20,7 @@ Download your dataset and load it into memory with these few lines of code:
 
 .. csv-table:: Public Datasets
    :file: public_datasets.csv
-   :widths: 30, 40, 10, 10, 10
+   :widths: 30, 40, 10, 10, 10, 10, 10, 10, 10
```

Comment:
Weird, on my local machine this raises a warning/error.
The widths are actually percentage values so they should add up to 100.

Reply:
should be fixed now?